README.md

```diff
@@ -64,6 +64,9 @@ If you are unable to do this, you can set `NO_MSRS=1` in your Unix environment
 when building Krun (see later), but be aware that this degrades the quality of
 the resulting benchmarking numbers.
 
+This version of Krun supports versions >= 4.9.88 of the custom kernel. Any
```

Comment:
Can we rephrase this to something a bit clearer. Maybe something like "Krun works on any version of the kernel if `NO_MSRS=1` is set. If a suitable version of the Krun custom kernel (>= 4.9.88) is used, the MSRs are also available"?

Reply:
That's much better. I did struggle a bit wording this (long day)...

Reply:
Fixed